File: java_SDK/PayHubSDK/src/main/java/com/payhub/ws/model/Merchant.java
package com.payhub.ws.model;

import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL) 
public class Merchant {
	private static final long serialVersionUID = 8527142639482598808L;

	public static final String TERMINAL_ID_FIELD = "terminal_id";

	public static final String ORGANIZATION_ID_FIELD = "organization_id";
	
    public int organization_id;

    public int terminal_id;

    private String api_username;

    private String api_password;

	public int getOrganization_id() {
		return organization_id;
	}

	public void setOrganization_id(int organization_id) {
		this.organization_id = organization_id;
	}

	public int getTerminal_id() {
		return terminal_id;
	}

	public void setTerminal_id(int terminal_id) {
		this.terminal_id = terminal_id;
	}

	public String getApi_username() {
		return api_username;
	}

	public void setApi_username(String api_username) {
		this.api_username = api_username;
	}

	public String getApi_password() {
		return api_password;
	}

	public void setApi_password(String api_password) {
		this.api_password = api_password;
	}
    
}

File: Payhub - SDK/php_SDK/com/payhub/ws/model/Merchant.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 23/07/2015
 * Time: 15:30
 */
class Merchant
{
    public static $TERMINAL_ID_FIELD = "terminal_id";

    public static $ORGANIZATION_ID_FIELD = "organization_id";

    public $organization_id;

    public $terminal_id;

    private $api_username;

    private $api_password;

    /**
     * @return mixed
     */
    public function getOrganizationId()
    {
        return $this->organization_id;
    }

    /**
     * @param mixed $organization_id
     */
    public function setOrganizationId($organization_id)
    {
        $this->organization_id = $organization_id;
    }

    /**
     * @return mixed
     */
    public function getTerminalId()
    {
        return $this->terminal_id;
    }

    /**
     * @param mixed $terminal_id
     */
    public function setTerminalId($terminal_id)
    {
        $this->terminal_id = $terminal_id;
    }

    /**
     * @return mixed
     */
    public function getApiUsername()
    {
        return $this->api_username;
    }

    /**
     * @param mixed $api_username
     */
    public function setApiUsername($api_username)
    {
        $this->api_username = $api_username;
    }

    /**
     * @return mixed
     */
    public function getApiPassword()
    {
        return $this->api_password;
    }

    /**
     * @param mixed $api_password
     */
    public function setApiPassword($api_password)
    {
        $this->api_password = $api_password;
    }

    public static function fromArray($data){
        if(!is_null($data)) {
            $merchant = new Merchant();
            foreach ($data as $key => $value) {
                if (property_exists(get_class($merchant), $key)) {
                    $merchant->{$key} = $value;
                }
            }
            return $merchant;
        }
    }
}